<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>172</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <widget class="QMenu" name="menuQuestionList">
     <property name="title">
      <string>QuestionList</string>
     </property>
     <addaction name="actionAdd"/>
     <addaction name="actionInsert"/>
     <addaction name="actionDelete"/>
     <addaction name="separator"/>
     <addaction name="actionEdit"/>
     <addaction name="actionEdit_All"/>
    </widget>
    <addaction name="actionConsole"/>
    <addaction name="actionAuto_generate_Save_File"/>
    <addaction name="menuQuestionList"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="text">
    <string>Console</string>
   </property>
  </action>
  <action name="actionAuto_generate_Save_File">
   <property name="text">
    <string>Auto-generate Save File</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add...</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>Insert...</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete...</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit Current</string>
   </property>
  </action>
  <action name="actionEdit_All">
   <property name="text">
    <string>Edit All...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
